openedx/core/djangoapps/waffle_utils/__init__.py

```diff
@@ -509,10 +71,16 @@ def is_enabled(self, course_key=None):  # pylint: disable=arguments-differ
                 outside the context of any course.
         """
         if course_key:
-            assert isinstance(course_key, CourseKey), (
-                "Provided course_key '{}' is not instance of CourseKey.".format(course_key)
+            assert isinstance(
+                course_key, CourseKey
+            ), "Provided course_key '{}' is not instance of CourseKey.".format(
+                course_key
+            )
+        is_enabled_for_course = self._get_course_override_value(course_key)
```

Comment:
`is_enabled_for_course` should be `override_value_for_course`.  It may be enabled or disabled.